actions/apps.ts

```diff
@@ -17,7 +17,7 @@ import {makeCallErrorResponse} from 'utils/apps';
 import {localizeAndFormatMessage, localizeMessage} from 'utils/utils';
 import {t} from 'utils/i18n';
 
-export function doAppCall<Res=unknown>(call: AppCallRequest, type: AppCallType): ActionFunc {
+export function doAppCall<Res=unknown>(call: AppCallRequest, type: AppCallType, _intl?: any): ActionFunc { // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Should we use it? This could be an improvement on component bindings since they will use the component intl, which from what I understand, is more reliable. And for commands or other situations, we can keep using the intlShim.

Finally, it will allow us to use everywhere the formatMessage, which might be more comprehensible and better tailored for the i18n extract.

Reply:
Using it now